tracer/test/test-applications/integrations/Samples.OpenTelemetrySdk/Samples.OpenTelemetrySdk.csproj

```diff
@@ -3,11 +3,13 @@
     <!-- System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.1-->
     <!-- https://andrewlock.net/stop-lying-about-netstandard-2-support/-->
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
-  </PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">1.3.1</ApiVersion>
+    <DefineConstants Condition="'$(ApiVersion)'>='1.2.0'">$(DefineConstants);OTEL_1_2</DefineConstants>
+    </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="1.3.1" />
-    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.3.1" />
-    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.3.1" />
+    <PackageReference Include="OpenTelemetry" Version="$(ApiVersion)" />
```

Comment:
🤔 also, we follow the version of OpenTelemetry.API but here we test the OpenTelemetry nuget. I suppose they are link but feels weird.

Reply:
Yeah, they're all in sync and transitive packages. I just used the OtelAPI package as that's what Zach had used, but any of them are fine tbh, I'm not passionate one way or another